appconfig/settings/wdb.q

```diff
@@ -0,0 +1,16 @@
+\d .finspace
+
+database:getenv[`KDBDATABASETRADE];
+hdbclusters:enlist `$"hdb";
+
+notifyhdb:{[h;d]
```

Comment:
This function can be deleted. We won't be notifying the hdb, we'll be starting a new one.

Same with .finspace.hdbclusters (line 3) (I think, double check there are no other uses other than in notifyhdb)

And same with .finspace.hdbmessage (line 11)